components/Builder.vue

```diff
@@ -103,15 +128,60 @@
           :features="component.features.data"
         />
       </div>
+
+      <div
+        class="background-gradient-lines"
+        v-else-if="typeOf(component) == 'clubs'"
+      >
+        <Clubs
+          :clubs="component.clubs.clubs"
+          :color="component.clubs.color"
+        />
+      </div>
+
+      <div
+        class="background-gradient-lines"
+        v-else-if="typeOf(component) == 'description'"
+      >
+        <Description
+          :quotes="component.description.quotes"
+        />
+      </div>
+
+      <div
+        class="background-gradient-lines"
+        v-else-if="typeOf(component) == 'social'"
+      >
+        <Social
+          :facebook="component.social.facebook"
+          :instagram="component.social.instagram"
+          :snapchat="component.social.snapchat"
+          :tiktok="component.social.tiktok"
+          :mail="component.social.mail"
+        />
+      </div>
+
+      <div
+        class="background-gradient-lines"
+        v-else-if="typeOf(component) == 'trombinoscope'"
+      >
+        <Trombinoscope
+          :title="component.trombinoscope.title"
+          :members="component.trombinoscope.members"
+        />
+      </div>
+
       <p v-else>-</p>
     </div>
   </div>
 </template>
 
 <script>
+import SocialNetworks from '~/components/Social';
 import GraphScore from '~/components/GraphScore';
+import Trombinoscope from '~/components/Trombinoscope';
 export default {
```

Comment:
I'm not sure if these imports are required, there are already components like `Carousel` that are used without being imported.